openassessment/templates/openassessmentblock/staff/oa_staff_incomplete_waiting.html

```diff
@@ -0,0 +1,31 @@
+{% load i18n %}
+
+<li id="openassessment__staff-assessment" class="openassessment__steps__step step--staff-assessment">
+    <header class="step__header">
```

Comment:
I'm wondering if this header could be a template. It seems to be almost identical in each template, except for the label and an optional icon.